src/utils/base64.ts

```diff
@@ -22,6 +23,39 @@ const isFilePromise = (file: string | Buffer): Promise<boolean> =>
     });
   });
 
+const isUrl = (file: string | Buffer): boolean => {
```

Comment:
Is the type required to be `string | Buffer` here? Can it not be just `string` because of the type narrowing [here](https://github.com/weaviate/typescript-client/pull/252/files#diff-a7864de1a55729a51dfe33cb5499284e6094ebc383e7bcd7f13dfe32d46d5bfaR72-R75) by `isBuffer`?

Can we also change it to be a true type guard so `: file is string` rather than `: boolean`? wdyt?

Reply:
Makes 100% sense @tsmith023 
I just made the required changes, please have a look again